app/routes.py

```diff
@@ -138,4 +138,12 @@ def friends():
 @app.route('/logout', methods=['GET'])
 def logout():
     logout_user()
-    return redirect('/')
\ No newline at end of file
+    return redirect('/')
+	
+@app.route('/home', methods=['GET', 'POST'])
+def menu_add():
```

Comment:
instead of having this be part of /home, have it add a menu as part of a different route and redirect to the homepage url after it adds. otherwise this gets really messy with multiple endpoints pointing to the same page